src/app/globals.css

```
@@ -410,15 +432,29 @@ h3 {
   display: grid;
   gap: 12px;
   margin-top: 16px;
+  min-width: 0;
+  max-width: 100%;
 }
 
 .eventTitle {
   font-weight: 600;
+  overflow-wrap: anywhere;
+  word-break: break-word;
```

Stylelint (17.4.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)